File: Assignment 2/Q3.c
#include <stdio.h>
#include <math.h>
int main(){
    int a,b,c ;
    printf("Enter the sides of the triangle: ");
    scanf("%d %d %d", &a, &b, &c);
    if (a*a==sqrt(b*b+c*c) || b*b==sqrt(a*a+c*c) || c*c==sqrt(a*a+b*b)){
        printf("The triangle is a right angled triangle");
    }
    else{
        printf("The triangle is not a right angled triangle");
    }
    return 0;
}

File: Assignment 2/Q3.py
import math
print("Enter the sides of Triangle")
a = int(input())
b = int(input())
c = int(input())

if (a**2 == math.sqrt(b**2 + c**2) or 
    b**2 == math.sqrt(a**2 + c**2) or 
    c**2 == math.sqrt(a**2 + b**2)):
    print("The triangle is a right angled triangle")
else:
    print("The triangle is not a right angled triangle")
